tailwind.config.js

```diff
@@ -9,6 +9,7 @@ export default {
         secondary: '#ede8e1',
         customGreen: '#3b4a20',
         customLightGreen: '#4c602a',
+        veryLightGreen: '#c9cfbf',
```

Comment:
Praise: Nice use of tailwind.config.js I learned something TIL. I wasn't aware that you could add custom colors in the config file. That is cool 😎